packages/typespec-client-generator-core/test/package.test.ts

```diff
@@ -3091,17 +3055,429 @@ describe("typespec-client-generator-core: package", () => {
       op test(...Intersected): void;
       `);
       const method = getServiceMethodOfClient(runner.context.experimental_sdkPackage);
-      const documentMethodParam = method.parameters.find(
-        (x) => x.name === "documentTranslateContent"
-      );
+      const documentMethodParam = method.parameters.find((x) => x.name === "document");
       ok(documentMethodParam);
       strictEqual(documentMethodParam.kind, "method");
       const op = method.operation;
       ok(op.bodyParam);
       strictEqual(op.bodyParam.kind, "body");
-      strictEqual(op.bodyParam.name, "documentTranslateContent");
+      strictEqual(op.bodyParam.name, "testRequest");
       deepStrictEqual(op.bodyParam.correspondingMethodParams, [documentMethodParam]);
     });
+
+    it("anonymous model with @body should not be spread", async () => {
```

Comment:
Looks like we have to take the generated names for anonymous model, because it has to be used in the operation signature.

Reply:
yes.